web/src/beta/components/Resizable/index.tsx

```diff
@@ -33,24 +41,50 @@ const Resizable: React.FC<Props> = ({
   const BottomGutter = showBottomGutter ? <HorizontalGutter {...gutterProps} /> : null;
   const LeftGutter = showLeftGutter ? <VerticalGutter {...gutterProps} /> : null;
 
+  useEffect(() => {
+    if (size <= initialSize / 2) {
+      setMinimized(true);
+    } else {
+      setMinimized(false);
+    }
+  }, [direction, initialSize, size]);
+
+  const handleShowOriginalPanel = useCallback(() => {
+    setMinimized(false);
+    onInitializeSize();
+  }, [onInitializeSize]);
   return (
-    <StyledResizable direction={direction} size={size}>
-      {TopGutter}
-      {LeftGutter}
-      <Wrapper>{children}</Wrapper>
-      {RightGutter}
-      {BottomGutter}
-    </StyledResizable>
+    <>
+      {minimized ? (
+        <MinimizedWrapper direction={direction} onClick={handleShowOriginalPanel}>
+          <Icon icon={gutter == "end" ? "arrowRight" : "arrowLeft"} />
+        </MinimizedWrapper>
+      ) : (
+        <StyledResizable direction={direction} size={size} minSize={minSize}>
+          {TopGutter}
+          {LeftGutter}
+          <Wrapper>{children}</Wrapper>
+          {RightGutter}
+          {BottomGutter}
+        </StyledResizable>
+      )}
+    </>
   );
 };
 
-const StyledResizable = styled.div<Pick<Props, "direction" | "size">>`
+const StyledResizable = styled.div<Pick<Props, "direction" | "size" | "minSize">>`
   display: flex;
   align-items: stretch;
   flex-direction: ${({ direction }) => (direction === "vertical" ? "row" : "column")};
   width: ${({ direction, size }) => (direction === "horizontal" ? null : `${size}px`)};
   height: ${({ direction, size }) => (direction === "vertical" ? null : `${size}px`)};
   flex-shrink: 0;
+  resize: both;
```

Comment:
`resize` property would be created for another feature and it's not working since the element is `overflow: visible`. ( I read [mdn](https://developer.mozilla.org/en-US/docs/Web/CSS/resize#syntax) )
![image](https://github.com/reearth/reearth/assets/16318727/f8075c4c-6ba2-441a-9f90-a500dffc2385)

So It would be good to delete it.

Reply:
deleted